### PR TITLE
[ODS-4420] Address code review comments for .NET core branch

### PR DESCRIPTION
**Pull request by raghuramakrishna**

### CODE REVIEW

File: Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs [hunk 1]
```diff
@@ -16,9 +16,9 @@ namespace EdFi.Admin.DataAccess.Contexts
 {
     public abstract class UsersContext : DbContext, IUsersContext
     {
-        public const string ConnectionStringName = "EdFi_Admin";
-
 #if NETFRAMEWORK
+
```
Comment (jamessmckay) [inline Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs:20]: remove blank line

File: Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs [hunk 2]
```diff
@@ -36,7 +37,10 @@ public UsersContextFactory(IApiConfigurationProvider configurationProvider)
 
         public UsersContextFactory(IAdminDatabaseConnectionStringProvider connectionStringsProvider, DatabaseEngine databaseEngine)
         {
-            _connectionStringsProvider = connectionStringsProvider;
+
```
Comment (jamessmckay) [inline Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs:40]: remove blank line

File: Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs [hunk 3]
```diff
@@ -36,7 +37,10 @@ public UsersContextFactory(IApiConfigurationProvider configurationProvider)
...
+        Preconditions.ThrowIfNull(connectionStringsProvider, nameof(connectionStringsProvider));
+         Preconditions.ThrowIfNull(databaseEngine, nameof(databaseEngine));
+            _connectionStringsProvider = connectionStringsProvider;          
             _databaseEngine = databaseEngine;
```
Comment (jamessmckay) [inline Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs:44]:
```suggestion
        _connectionStringsProvider = Preconditions.ThrowIfNull(connectionStringsProvider, nameof(connectionStringsProvider));
        _databaseEngine =Preconditions.ThrowIfNull(databaseEngine, nameof(databaseEngine));
```

File: Application/EdFi.Ods.Admin/Contexts/IdentityContextFactory.cs [hunk 4]
```diff
@@ -43,7 +43,8 @@ public IdentityContext CreateContext()
             }
 
             throw new InvalidOperationException(
-                $"Cannot create an IUsersContext for database type {_databaseEngine.DisplayName}");
+                $"Cannot create an IIdentityContext  for database type
+                {_databaseEngine.DisplayName}");
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Admin/Contexts/IdentityContextFactory.cs:47]:
```suggestion
    $"Cannot create an IUsersContext for database type {_databaseEngine.DisplayName}");
```

File: Application/EdFi.Ods.Admin/Extensions/HttpContextExtensions.net.cs [hunk 5]
```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+#if NETFRAMEWORK
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Admin/Extensions/HttpContextExtensions.net.cs:5]:
```suggestion

#if NETFRAMEWORK
```

File: Application/EdFi.Ods.Admin/Security/IdentityProvider.cs [hunk 6]
```diff
@@ -26,10 +26,13 @@ public IdentityProvider(IIdentityContextFactory identityContextFactory)
...
-            var identityUserManager = GetIdentityUserManager();
-            var identityUser = identityUserManager.FindByEmail(userEmail);
-            return identityUser != null;
+            using (var identityUserManager = GetIdentityUserManager())
+            {
+                var identityUser = identityUserManager.FindByEmail(userEmail);
+                return identityUser != null;
+            }
         }
+
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Admin/Security/IdentityProvider.cs:35]: remove blank line

File: Application/EdFi.Ods.Admin/Security/IdentityProvider.cs [hunk 7]
```diff
@@ -157,12 +160,12 @@ public bool ConfirmEmailWithToken(string email, string token)
         {
             using (var identityUserManager = GetIdentityUserManager())
             {
-                IdentityResult result = null;
+
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Admin/Security/IdentityProvider.cs:163]: remove blank line

File: Application/EdFi.Ods.Admin/Security/IdentityProvider.cs [hunk 8]
```diff
@@ -157,12 +160,12 @@ public bool ConfirmEmailWithToken(string email, string token)
...
             using (var identityUserManager = GetIdentityUserManager())
             {
-                IdentityResult result = null;
+               
                 var identityUser = identityUserManager.FindByEmail(email);
 
                 if (identityUser != null)
-                {
-                    result = identityUserManager.ConfirmEmail(identityUser.Id, token);
+                {
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Admin/Security/IdentityProvider.cs:167]: remove extra whitespace

File: Application/EdFi.Ods.Admin/Security/IdentityProvider.cs [hunk 9]
```diff
@@ -157,12 +160,12 @@ public bool ConfirmEmailWithToken(string email, string token)
...
             {
-                IdentityResult result = null;
+               
                 var identityUser = identityUserManager.FindByEmail(email);
 
                 if (identityUser != null)
-                {
-                    result = identityUserManager.ConfirmEmail(identityUser.Id, token);
+                {               
+                   IdentityResult result = identityUserManager.ConfirmEmail(identityUser.Id, token);
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Admin/Security/IdentityProvider.cs:168]: var please

File: Application/EdFi.Ods.Features/Container/Installers/EdFiExtensionsInstaller.net.cs [hunk 10]
```diff
@@ -1,4 +1,8 @@
 #if NETFRAMEWORK
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Features/Container/Installers/EdFiExtensionsInstaller.net.cs:5]:
```suggestion
// SPDX-License-Identifier: Apache-2.0
// Licensed to the Ed-Fi Alliance under one or more agreements.
// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
// See the LICENSE and NOTICES files in the project root for more information.

#if NETFRAMEWORK
```

File: Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs (hunk 1 above)
Comment (syednoorulla) [reply Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs]: Fixed

File: Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs (hunk 2 above)
Comment (syednoorulla) [reply Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs]: fixed

File: Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs (hunk 3 above)
Comment (syednoorulla) [reply Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs]: Fixed

File: Application/EdFi.Ods.Admin/Contexts/IdentityContextFactory.cs (hunk 4 above)
Comment (syednoorulla) [reply Application/EdFi.Ods.Admin/Contexts/IdentityContextFactory.cs]: Fixed

File: Application/EdFi.Ods.Admin/Extensions/HttpContextExtensions.net.cs (hunk 5 above)
Comment (syednoorulla) [reply Application/EdFi.Ods.Admin/Extensions/HttpContextExtensions.net.cs:6]: Fixed

File: Application/EdFi.Ods.Admin/Security/IdentityProvider.cs (hunk 6 above)
Comment (syednoorulla) [reply Application/EdFi.Ods.Admin/Security/IdentityProvider.cs:35]: Fixed

File: Application/EdFi.Ods.Admin/Security/IdentityProvider.cs (hunk 7 above)
Comment (syednoorulla) [reply Application/EdFi.Ods.Admin/Security/IdentityProvider.cs]: Fixed

File: Application/EdFi.Ods.Admin/Security/IdentityProvider.cs (hunk 8 above)
Comment (syednoorulla) [reply Application/EdFi.Ods.Admin/Security/IdentityProvider.cs:166]: Did not find any extra whitespace.

File: Application/EdFi.Ods.Admin/Security/IdentityProvider.cs (hunk 9 above)
Comment (syednoorulla) [reply Application/EdFi.Ods.Admin/Security/IdentityProvider.cs]: fixed

File: Application/EdFi.Ods.Features/Container/Installers/EdFiExtensionsInstaller.net.cs (hunk 10 above)
Comment (syednoorulla) [reply Application/EdFi.Ods.Features/Container/Installers/EdFiExtensionsInstaller.net.cs]: Fixed

File: Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs [hunk 11]
```diff
@@ -16,9 +16,8 @@ namespace EdFi.Admin.DataAccess.Contexts
 {
     public abstract class UsersContext : DbContext, IUsersContext
     {
-        public const string ConnectionStringName = "EdFi_Admin";
-
 #if NETFRAMEWORK
+        public const string ConnectionStringName = "EdFi_Admin";
```
Comment (jamessmckay) [inline Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs:20]:
```suggestion
        public const string ConnectionStringName = "EdFi_Admin";
        
```

File: Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs [hunk 12]
```diff
@@ -36,8 +37,8 @@ public UsersContextFactory(IApiConfigurationProvider configurationProvider)
 
         public UsersContextFactory(IAdminDatabaseConnectionStringProvider connectionStringsProvider, DatabaseEngine databaseEngine)
         {
-            _connectionStringsProvider = connectionStringsProvider;
-            _databaseEngine = databaseEngine;
+       _connectionStringsProvider = Preconditions.ThrowIfNull(connectionStringsProvider, nameof(connectionStringsProvider));
+        _databaseEngine =  Preconditions.ThrowIfNull(databaseEngine, nameof(databaseEngine));
```
Comment (jamessmckay) [inline Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs:41]: fix this formatting please

File: Application/EdFi.Ods.Admin/Security/IdentityProvider.cs (hunk 8 above)
Comment (jamessmckay) [reply Application/EdFi.Ods.Admin/Security/IdentityProvider.cs:166]: the whitespace is still showing up, if you refomat the document it should clear it

File: Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs (hunk 11 above)
Comment (raghuramakrishna) [reply Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs:20]: Fixed

File: Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs (hunk 12 above)
Comment (raghuramakrishna) [reply Application/EdFi.Admin.DataAccess/Contexts/UsersContextFactory.cs:41]: Fixed

**Pull request merged**